src/modules/colorPicker/ColorPickerUI/Controls/ColorPickerControl.xaml.cs

```diff
@@ -64,9 +64,9 @@ private static void SelectedColorPropertyChanged(DependencyObject d, DependencyP
             ((ColorPickerControl)d)._ignoreHexChanges = true;
             ((ColorPickerControl)d)._ignoreRGBChanges = true;
             ((ColorPickerControl)d).HexCode.Text = ColorToHex(newColor);
-            ((ColorPickerControl)d).RTextBox.Text = newColor.R.ToString(CultureInfo.InvariantCulture);
-            ((ColorPickerControl)d).GTextBox.Text = newColor.G.ToString(CultureInfo.InvariantCulture);
-            ((ColorPickerControl)d).BTextBox.Text = newColor.B.ToString(CultureInfo.InvariantCulture);
+            ((ColorPickerControl)d).RNumberBox.Text = newColor.R.ToString(CultureInfo.InvariantCulture);
```

Comment:
Would be nice to store that `((ColorPickerControl)d)` in `colorPickerControl` variable so we avoid this excessive casting in this method

Reply:
@niels9001 
if you want I can push the fix for this.

Reply:
That would be nice :)!

Reply:
Done.